### ISSUE TITLE
Configured with --fmemopen : compile option -fmemopen

### ISSUE THREAD

**Issue by riesebie** [neomutt/neomutt#839]
```
LC_ALL=C neomutt -v | grep -B1 fmemopen                                                                                                                         
                                                                                                                                                                                           
Configure options: --sysconfdir=/etc --with-mailpath=/var/mail --gpgme --fmemopen --notmuch \
--logging --lua --mixmaster --with-ui=ncurses --gss --gnutls --sasl --lmdb                     
--                                                                                                                                                                                         
Compile options:                                                                                                                                                                           
  +bkgdset +color +curs_set +debug +fcntl -flock -fmemopen +futimens
```

**Comment by gahr** [comment]
Yes, fmemopen is unconditionally turned off.

```
###############################################################################
# fmemopen(3)
if {[get-define want-fmemopen] && [cc-check-functions fmemopen open_memstream]} {
  # Temporarily disable fmemopen, due to this bug:
  # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=834408
  define USE_FMEMOPEN 0
} else {
  define USE_FMEMOPEN 0
}
```

**Comment by riesebie** [comment]
Hmm, so it shouldn't be possible to configure.autosetup --fmemopen.

**Comment by riesebie** [comment]
Are there any thoughts to fix it? Maybe there is no big advantage and we can think about throwing it away?

**Comment by riesebie** [comment]
If fmemopen is turned off no one can test it. Is torify on i686 the only crashable environment? Does one know whether other distros than Debian based (SuSe, Fedora, CentOS, Slack, Gentoo etc) use that feature either with mutt or neomutt? Don't know exactly wheater *BSD uses it.... Escalation can point to a solution, though. For me I cant't remember of a big performance spot running fmemopen.

**Comment by gahr** [comment]
I do use fmemopen (via a modified auto.def) and it’s working fine. I don’t use tor though, so no idea how it behaves there.
If it was me, I’d just restore the flag to do what it says it does, and leave it off by default.
Thoughts?

**Comment by riesebie** [comment]
> If it was me, I’d just restore the flag to do what it says it does, and leave it off by default.
Well, I'd prefer this for now too. Packagers can decide whether they want to enable it or not. But it gives potential TOR dev's the chance to think about the use of fmemopen.

**Comment by flatcap** [comment]
Fixed by 3f42b5f
